### PR TITLE
refactor(server): review classes and removed garbage files

### PR DESCRIPTION
**Pull request by reynan-dev**
## Ticket

**Title:** ETQDev, j’ai remove toutes les methodes et classes inutiles dans le serveur
**Notion Link:** https://www.notion.so/jkergal/ETQDev-j-ai-remove-toutes-les-methodes-et-classes-inutiles-dans-le-serveur-5e7cd07c279b438983e3a4cdd55ba4c5?pvs=4

## Description

- Removed files linked on FileProject and IsSandpackFiles decorator
- Passed Errors responsibilities from Services to Resolvers
- Alphabetic sort methods on services and resolvers classes.
- Moved codenaute cli to utils

## Guidelines

🔗 Link your PRs to their related Notion tickets
✍️ Use the commit convention to your pull request title
📃 If your code affects the build, update `README.md`
🚫 Do not merge a PR until all comments are resolved
🗑 Remove your branch after merging

### CODE REVIEW

File: server/src/resolvers/ProjectResolver.ts [hunk 1]
```diff
@@ -65,118 +100,57 @@ export class ProjectResolver {
...
-
-		return this.ProjectServices.create({
-			name: name,
-			owner: member,
-			isTemplate: isTemplate,
-			isPublic: isPublic,
-			sandpackTemplate: sandpackTemplate,
-			files: files
-		});
+	async getAllProjectsPublicProjects(): Promise<Project[]> {
```
Comment (jkergal) [inline server/src/resolvers/ProjectResolver.ts:103]: 2x "projects" in the name?

**Review by jkergal — COMMENTED** [review]
Did you make sure that this PR doesn't break anything?
For example, if you change something in the graphQL schemas on the server side, please make sur that you don't need to modify something in the front, relanch codegen to update queries and mutation etc....

File: server/src/resolvers/ProjectResolver.ts (hunk 1 above)
Comment (reynan-dev) [reply server/src/resolvers/ProjectResolver.ts:103]:
```suggestion
	async getAllPublicProjects(): Promise<Project[]> {
```

**Comment by reynan-dev** [comment]
> Did you make sure that this PR doesn't break anything? For example, if you change something in the graphQL schemas on the server side, please make sur that you don't need to modify something in the front, relanch codegen to update queries and mutation etc....

Usually, all methods follow the client structure and pass through all tests on the server side. But before merging it, we can test run all applications and test every method changed or removed.

**Comment by jkergal** [comment]
> > Did you make sure that this PR doesn't break anything? For example, if you change something in the graphQL schemas on the server side, please make sur that you don't need to modify something in the front, relanch codegen to update queries and mutation etc....
> 
> Usually, all methods follow the client structure and pass through all tests on the server side. But before merging it, we can test run all applications and test every method changed or removed.

Yes but if you changed some naming, you need to update client GraphQL schema by running codegen... Sometime, before you make some changes i've got some broke feature because of it, don't forget to check it ! :)

**Pull request merged**